experimental/middleware/middleware.go

```diff
@@ -4,11 +4,33 @@
 package experimental
 
 import (
+	"embed"
 	"net/http"
 
 	"github.com/corazawaf/coraza/v3"
 )
 
+//go:embed error_template.html error_template.html
```

Comment:
```suggestion
//go:embed error_template.html interruption_template.html
```

That's just to fix this. But I suggest changing to "default_error.html" and "default_interruption.html", pending on the next comment.